entity/contents/contents.cc

```diff
@@ -30,12 +30,7 @@ Contents::Contents() = default;
 Contents::~Contents() = default;
 
 Rect Contents::GetBounds(const Entity& entity) const {
-  const auto& transform = entity.GetTransformation();
-  auto points = entity.GetPath().GetBoundingBox()->GetPoints();
-  for (uint i = 0; i < points.size(); i++) {
-    points[i] = transform * points[i];
-  }
-  return Rect::MakePointBounds({points.begin(), points.end()});
+  return entity.GetTransformedPathBounds();
```

Comment:
Not to worry about here, but when we implement vertices based contents we'll have to handle this differently

Reply:
Yeah, GetBounds is virtual and some contents will need overrides.